src/extension/providers/mcp_server_definition_provider.ts

```diff
@@ -35,6 +35,10 @@ export class DartMcpServerDefinitionProvider implements vs.McpServerDefinitionPr
 		const args = ["mcp-server"];
 		if (this.dartCapabilities.mcpServerRequiresExperimentalFlag)
 			args.push("--experimental-mcp-server");
```

Comment:
This experimental flag is not required since `3.9.0-293` so update `mcpServerRequiresExperimentalFlag` to only provide this for versions up until then.